bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -323,12 +323,22 @@ def VecOps(self):
     # Overload RDF namespace
     @property
     def RDF(self):
+        import numpy
         ns = self._fallback_getattr('RDF')
         try:
             # Inject FromNumpy function
             from libROOTPythonizations import MakeNumpyDataFrame
-            ns.FromNumpy = MakeNumpyDataFrame
 
+            # Make a copy of the arrays that have strides
+            # TODO a cleaner fix 
+            def MakeNumpyDataFrameCopy(pyvec):    
```

Comment:
Something I'm still not sure about is the signature of the function. The factory defined [in RNumpyDS.hxx](https://github.com/root-project/root/blob/aa3bff2fd223b46b448ed480a14fa567ef9650b7/bindings/pyroot/pythonizations/inc/RNumpyDS.hxx#L225-L226) accepts two arguments. Now to be fair, I've never seen usage of the second argument and the class doesn't document it. So probably this patch is still fine (it indeed passes all the tests we have!), but something to be investigated further at some point

Reply:
ok, so for now I will leave it as it is